### DISCUSSION TITLE
Whitespace is being included in Pair unexpectedly

### DISCUSSION THREAD

**Discussion by jonah-saltzman** [pest-parser/pest#951] [Q&A]
For the following `.pest` file:
```
WHITESPACE = _{ " " }
COLOR      = { "blue" | "red" | "green" }
NUM_CUBES  = { ASCII_DIGIT+ }
SET_ELE    = { NUM_CUBES ~ COLOR }
```
The input `2 green` parsed against the `SET_ELE` rule produces:
```
Pair { rule: SET_ELE, span: Span { str: "2 green", start: 0, end: 7 }, inner: [Pair { rule: NUM_CUBES, span: Span { str: "2 ", start: 0, end: 2 }, inner: [Pair { rule: WHITESPACE, span: Span { str: " ", start: 1, end: 2 }, inner: [] }] }, Pair { rule: COLOR, span: Span { str: "green", start: 2, end: 7 }, inner: [] }] }
```
Notice that the `NUM_CUBES` Pair's span is `"2 "` rather than what I expected, `"2"`. Any idea what the issue is?

**Comment by tomtau** [comment]
Maybe it hit this bug: https://github.com/pest-parser/pest/issues/396 ?
The fix for it is feature-guarded with "grammar-extras" in order to keep the old behaviour under default (as it was unclear whether any crates depend on that old behaviour)
Did you specify your dependency as follows ?
```
pest_derive = { version = "2.7", features = ["grammar-extras"] }
```
